app/main/views/notifications.py

```diff
@@ -53,7 +54,9 @@ def user_research_consent():
     cookie_name = 'seen_user_research_message'
 
     if cookie_name not in request.cookies:
-        additional_headers = {'Set-Cookie': "{}=yes; Path=/".format(cookie_name)}
+        expiry_date = datetime.datetime.now() + datetime.timedelta(90)
+        expiry_date = expiry_date.strftime("%a, %d-%b-%Y %H:%M:%S GMT")
```

Comment:
I don't see why we need this any more: if they haven't opted in, then surely we can carry on showing the message - until they click Close?

Reply:
Hmm, yeah it is a bit weird to keep showing the banner if they've specifically click Save having opted out.

And while it might make sense to keep showing it until they've posted the form, let's not make this more complicated by adding that restriction here.

It's fine as it is.